terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/WebDriverOperations.java

```diff
@@ -44,6 +48,15 @@ public void setDefaultTimeoutForImplicitlyWait(
     }
 
     /**
+     * Get the text (display value) set for the specified element.
+     * @param by Identifier to look for elements
+     * @return And returns the text (display value)
+     */
+    public String getText(By by) {
+    	return webDriver.findElement(by).getText();
+    }

```

Comment:
Certainly I think `WebDriverOperations#getApServerVersion` is not a completed method.
However, I think implementation which has a method that acquires server name but does not have a method that acquires server vaersion is not a nice implementation.
Please tell me how do you think?